mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -115,6 +115,15 @@ typedef std::unique_ptr<float[]> ModelMatrix;
 
 }  // namespace
 
+// Functions to help in calculation of normals
+namespace Normals {
```

Comment:
nit: having these helper functions in an anonymous namespace is good practice when we don't intend for anyone outside this file to ever call them.  Otherwise, I think reverting to have them be inside the calculator class would be preferable.